src/main/java/atreides/command/ErrorCommand.java

```diff
@@ -0,0 +1,33 @@
+package atreides.command;
+
+import atreides.task.TaskList;
+import atreides.ui.Storage;
+import atreides.ui.Ui;
+
+public class ErrorCommand implements Command {
+    private final String error;
+
+    public ErrorCommand(String error) {
+        this.error = error;
+    }
+
+    /**
+     * Ui will show the error message
+     * @param tasks
+     * @param ui
+     * @param storage
```

Comment:
Although these params aren't used, should they still have descriptions for what would usually be the input?